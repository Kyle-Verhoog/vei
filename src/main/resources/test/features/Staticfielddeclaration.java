public class Staticfielddeclaration {
  protected static int x;
}