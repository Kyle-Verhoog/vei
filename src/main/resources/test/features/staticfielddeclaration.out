PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PROTECTED ┃ protected
STATIC ┃ static
INT ┃ int
IDENTIFIER ┃ x
; ┃ ;
} ┃ }
